jenkinsbuilder/weekly-scan.py

```diff
@@ -111,22 +110,17 @@
 
         job = bs.put(json.dumps(data))
         # Initializing Database entries
-        project, created = Project.objects.get_or_create(
-            name=project_name,
-            target_file_link=form_targetfile_link(
-                entry["git-url"],
-                entry["git-path"],
-                entry["git-branch"],
-                entry["target-file"]
-            )
-        )
-        project.target_file_link = form_targetfile_link(
-            entry["git-url"],
-            entry["git-path"],
-            entry["git-branch"],
-            entry["target-file"]
+        project = Project.objects.get(
+            name=project_name
         )
-        project.save()
+        if not project:
+            print(
+                str.format(
+                    "Skipping {}, as project object was not found",
+                    project_name
+                )
+            )
```

Comment:
I could not understand why you would spread print statement over 6 lines..
```
print(str.format("Skipping {}, as project object was not found.", project_name))

was just fine too..

the print format used in PR is actually less-readable than one-line.
```
@mohammedzee1000